internal/distro/fedora/manifests.go

```diff
@@ -460,3 +460,48 @@ func bootISOPipeline(m *manifest.Manifest,
 	p.Filename = filename
 	return p
 }
+
+func tarBootISOTreePipeline(m *manifest.Manifest,
+	buildPipeline *manifest.Build,
+	anacondaPipeline *manifest.Anaconda,
+	treePipelineName,
```

Comment:
We should reference pipelines by reference, not by name, that way we get type safety.